model/riscv_pmp_control.sail

```diff
@@ -92,6 +92,21 @@ function pmpAddrRange(cfg: Pmpcfg_ent, pmpaddr: xlenbits, prev_pmpaddr: xlenbits
   }
 }
 
+/* Check MMl=1 and mmwp=0 behaviour*/
+
+val MMLcheck: (AccessType(ext_access_type)) -> bool effect {rreg}
```

Comment:
I would call this something else, it's a check for a region that is unmatched and MML is enabled. So maybe rename it `unmatchedMmlCheck`?